app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReportsController < ApplicationController
+  before_action :logged_in_user, only: [:new, :create, :destroy]
+
+  def new
+    render "reports/new"
+  end
+
+  def create
+    report = current_user.reports.build(report_params)
+    if @report.save
+      flash[:success] = "Created"
+      render "reports/reports"
```

Comment:
em dùng `redirect_to` nhé
khi report được lưu thành công, em chuyển người dùng đến trang liệt kê toàn bộ report nhé